wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -54,9 +56,100 @@ public static function getSubscribedEvents() {
         ['setCollectionCampAddress', 9],
         ['setEventVolunteersAddress', 8],
       ],
+      '&hook_civicrm_tabset' => 'collectionCampTabset',
     ];
   }
 
+  /**
+   *
+   */
+  public static function collectionCampTabset($tabsetName, &$tabs, $context) {
+    if (!self::isViewingCollectionCamp($tabsetName, $context)) {
+      return;
+    }
+
+    // URL for the Logistics tab.
+    $logisticsUrl = \CRM_Utils_System::url(
+      "wp-admin/admin.php?page=CiviCRM&q=civicrm%2Flogistics-coordination#",
+    );
+
+    // URL for the Dispatch tab.
+    $vehicleDispatch = \CRM_Utils_System::url(
+      "wp-admin/admin.php?page=CiviCRM&q=civicrm%2Fcamp-vehicle-dispatch-data",
+    );
+
+    // URL for the camp outcome tab.
+    $campOutcome = \CRM_Utils_System::url(
+      "wp-admin/admin.php?page=CiviCRM&q=civicrm%2Fadmin-camp-outcome-form",
+    );
```

Comment:
You will see a lot of code duplication that can be easily removed if we create an array